examples/confidential_computing/main.tf

```diff
@@ -0,0 +1,90 @@
+/**
+ * Copyright 2024 Google LLC
```

Comment:
I think we're missing a README.md file in the main directory

Reply:
Just added it, please take a look.